app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/OfflineEditFragment.java

```diff
@@ -896,4 +898,25 @@ public void onLongClick(int position) {
                 })
                 .show();
     }
+
+    private OpenFoodAPIService getApiServiceForProductType(int productType){
+
+        final int OPEN_FOOD_FACT = 5;
+        final int OPEN_BEAUTY_FACTS = 6;
+        final int OPEN_PET_FOOD_FACTS = 7;
+        final int OPEN_PRODUCTS_FACTS = 8;
+
+        switch (productType){
+            case OPEN_FOOD_FACT:
+                return new OpenFoodAPIClient(getActivity(), "https://ssl-api.openfoodfacts.org").getAPIService();
```

Comment:
https://world.openfoodfacts.org will work as well

Reply:
I'll change that.